### PR TITLE
Add new GSoC-2020 proposal for ATLAS geometry visualization

### PR DESCRIPTION
**Pull request by ric-bianchi**
Adding a new proposal for a visualization/detector-description/3d-graphics project in ATLAS.

### CODE REVIEW

File: _gsocproposals/2020/proposal_ATLAS-Visualization.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: GeoModelExplorer, interactive 3D geometry visualization - Development of a visualization tool to interactively explore the  geometry of a HEP detector
+layout: gsoc_proposal
+project: ATLAS
```
Comment (aflp91) [inline _gsocproposals/2020/proposal_ATLAS-Visualization.md]: Is there an ATLAS project described somewhere yet? If not, you have to add one there.

Comment (ric-bianchi) [reply _gsocproposals/2020/proposal_ATLAS-Visualization.md]: Thanks a lot for your commet, @aflp91! 
I didn't notice that this was the first proposal as "ATLAS". 

I will add that now.

Comment (ric-bianchi) [reply _gsocproposals/2020/proposal_ATLAS-Visualization.md]: Done.
Please, let me know if further additions or changes are needed.
Thanks, -- Riccardo.

**Comment by ric-bianchi** [comment]
The CI tests failed, but I don't see errors triggered by my proposal page in the log. It looks like the errors are from links in other pages.

**Comment by EdwardMoyse** [comment]
I saw the same for Phoenix.

File: _gsocprojects/2020/project_ATLAS.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+project: ATLAS
+layout: default
+logo: ATLAS-Logo-Ref-RGB.png
```
Comment (xvallspl) [inline _gsocprojects/2020/project_ATLAS.md]: The logo has to be named "ATLAS-logo.png"

File: _gsocproposals/2020/proposal_ATLAS-Visualization.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+## Expected results
+
+The integration of a modern graphics engine, new software tools focusing on detector geometry, possibility to animate the views to better explore the volumes, the ability to export 3D data to common formats.
+
+## Requirements
+
+C++, Qt5 framework (3D graphics experience a bonus)
+
+
+## Mentors
```
Comment (xvallspl) [inline _gsocproposals/2020/proposal_ATLAS-Visualization.md]: All the mentors have to be in mentors.md

Comment (ric-bianchi) [reply _gsocproposals/2020/proposal_ATLAS-Visualization.md]: Done.

File: _gsocprojects/2020/project_ATLAS.md (hunk 2 above)
Comment (ric-bianchi) [reply _gsocprojects/2020/project_ATLAS.md]: Done.

**Comment by ric-bianchi** [comment]
I added the missing entry to the global mentors' list and updated the filename of the ATLAS logo in the ATLAS project page, as requested.

Thanks for the corrections, and have a good day.

Best, -- Riccardo.

**Pull request merged**